docs/proxy/residential_proxy/examples.md

```diff
@@ -0,0 +1,620 @@
+---
+title: Examples
+description: Learn how to connect to Apify's residential proxies from your application with Node.js (axios and got), Python 2 and 3 and PHP using code examples.
+paths:
+    - proxy/residential-proxy/nodejs-examples
+    - proxy/residential-proxy/python-examples
+    - proxy/residential-proxy/php-examples
+    - proxy/residential-proxy/examples
+---
+<!-- Watch out! This file is hard to read because our own "marked-tabs" language fences aren't syntax-highlighted -->
+
+# Connect to residential proxies
```

Comment:
Do we really need another huge batch of examples for residentials, when they are no different than any other proxy group in terms of connection?

I understood the idea about SERP, where we provided Google specific examples, but this is exactly the same as datacenter stuff.

I think here it should just say that `RESIDENTIAL` is just like any other proxy group and link to `How to set a proxy group` examples and `How to set proxy country` examples.